perses/app/relative_setup.py

```diff
@@ -122,27 +122,21 @@ def __init__(self,
             box dimensions for the complex phase
         solvent_box_dimensions: Vec(3), optional, default=None
             box dimensions for the solvent phase
-        map_strategy : 'str' default='geometry'
-            determines which map is considered the best and returned
-            options are 'geometry', 'random', 'weighted-random', 'return-all'
-            can be one of ['geometry', 'matching_criterion', 'random', 'weighted-random', 'return-all']
-            - core will return the map with the largest number of atoms in the core. If there are multiple maps with the same highest score, then `matching_criterion` is used to tie break
-            - geometry uses the coordinates of the molB oemol to calculate the heavy atom distance between the proposed map and the actual geometry
-            this can be vital for getting the orientation of ortho- and meta- substituents correct in constrained (i.e. protein-like) environments.
-            this is ONLY useful if the positions of ligand B are known and/or correctly aligned.
-            - matching_criterion uses the `matching_criterion` flag to pick which of the maps best satisfies a 2D requirement.
-            - random will use a random map of those that are possible
-            - weighted-random uses a map chosen at random, proportional to how close it is in geometry to ligand B. The same as for 'geometry', this requires the coordinates of ligand B to be meaninful
-            - return-all BREAKS THE API as it returns a list of dicts, rather than list. This is intended for development code, not main pipeline.
         remove_constraints : bool, default=False
             if hydrogen constraints should be constrained in the simulation
             default is False, so no constraints removed, but 'all' or 'not water' can be used to remove constraints.
         use_given_geometries : bool, default False
             whether to extract the positions of ligand B and set the unique_new atom positions deterministically;
             if True, `complex` must be in `phases` and .sdf or .mol2 file of ligand must be provided
+        given_geometries_tolerance : simtk.unit.Quantity with units of length, default=0.2*angstrom
+            If use_given_geometries=True, use this tolerance for identifying mapped atoms
         """
         from openeye import oechem
 
+        # TODO: Refactor this class into initializer that configures options and factory methods that use them.
+        # QUESTION: Why do we add these as private object attributes?
+        #   If we want users to be able to modify them, they should not be private.
+        #   Also, these are all used immediately within __init__, so it doesn't make sense to store them this way.
```

Comment:
Yeah, I agree. It should be better to separate these into different methods that use the attributes.

Reply:
I believe @mikemhenry is working on refactoring this file right now for 0.10.0